src/WebAssert.php

```diff
@@ -219,7 +219,7 @@ public function responseHeaderNotContains($name, $value)
      */
     public function responseHeaderMatches($name, $regex)
     {
-        $actual = $this->session->getResponseHeader($name);
+        $actual = (string) $this->session->getResponseHeader($name);
```

Comment:
Why this (and similar type-casted places) is needed?

Reply:
https://3v4l.org/hOfGr/rfc#vgit.master

https://php.watch/versions/8.1/internal-func-non-nullable-null-deprecation